src/test/java/com/unbappd/enterprise/EnterpriseApplicationTests.java

```diff
@@ -48,13 +48,13 @@ private void whenUserSearchesWithPlaceID9(){
     }
     private void thenReturnReviewsWithPlaceID9(){
         String returnedReviewText = review.getReviewText();
-        int checkPlacesID = review.getPlacesId();
+        String checkPlacesID = review.getPlacesId();
         assertEquals("A fabulous beer", returnedReviewText);
         assertEquals(9,checkPlacesID);
     }
 
     @Test
-    void saveNewReview_validateNewReviewByReturningReviewText() {
+    void saveNewReview_validateNewReviewByNewReviewText() {
```

Comment:
I feel the variable name up there is suuuuper long, anyway to shortened it? Also I changed the CheckplacesID to string and changed it in the controllers.

Reply:
1) Method names in test cases should be long and descriptive.  That's a best practice.  When a test fails, you can quickly see which test, and why.
2) What is the added value by changing int to String?